style/src/test/java/org/jline/style/StyleExpressionTest.groovy

```diff
@@ -26,7 +26,7 @@ class StyleExpressionTest
     @Before
     void setUp() {
         super.setUp()
-        this.underTest = new StyleExpression(new StyleResolver(source, 'test'))
+        this.underTest = new StyleExpression()
```

Comment:
not exactly sure why you did this, this test probably needs; as I mentioned some validation of how the source is applied, which is what the super-class sets up for these tests.  Presently though the source isn't important, but thats a missing item to resolve for the expression stuff IMO.